src/common/components/notifications/_index.scss

```diff
@@ -65,8 +65,75 @@
     }
   }
 
+  .status-button-container {
+    border-bottom: 1px solid #f5f5f5;
+    display: flex;
+    justify-content: space-between;
+    padding: 10px 1px 20px 10px;
+
+    @include themify(day) {
+      border-bottom: 1px solid $white-three;
```

Comment:
Maybe better to change just color? `border-color: $white-three`. Same for dark mode